interface/patient_file/summary/demographics.php

```diff
@@ -161,14 +163,11 @@ function image_widget($doc_id, $doc_catg)
 <html>
 
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" type="text/css" href="../../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
-<script type="text/javascript" src="../../../library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-7-2/index.js"></script>
-<script type="text/javascript" src="../../../library/js/common.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+
+    <?php Header::setupHeader(['common']); ?>
+    <link rel="stylesheet" type="text/css" href="../../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
+    <script type="text/javascript" src="../../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+
```

Comment:
@sjpadgett , @bradymiller 
Hi.
The nice improvement of single asset for all the pages makes problems to fancy box library, fancy box doesn't support a last version of Jquery. So all the popups in the page (like birthday popup) are failed.

The version of Jquery is working with fancybox is 'jquery-min-1-7-2'.

Thanks
Amiel

Reply:
Okay, I'll resolve somehow. Either the fancybox go or a compatibility fix. Should have something later today. Thanks for bringing to attention.

Reply:
Thanks.
As I remember in the project has fancybox 1 and it deprecated before a few years...